### PR TITLE
data2bids events json update

### PR DESCRIPTION
**Pull request by DidiLamers**
- changed CapModelName to CapManufacturersModelName for the eeg events json, as that is the standard according to the BIDS spec

### CODE REVIEW

File: data2bids.m [hunk 1]
```diff
@@ -1542,7 +1542,7 @@
     begsample                   = table2array(events_tsv(:,{'begsample'}));
     endsample                   = table2array(events_tsv(:,{'endsample'}));
     onset                       = (begsample-1)./hdr.Fs;
-    duration                    = (endsample-begsample+1)./hdr.Fs; 
+    duration                    = (endsample-begsample)./hdr.Fs;
```
Comment (robertoostenveld) [inline data2bids.m:1545]: Can you please revert this? If an event lasts from sample N to sample N+1, I would say that its duration is 2 samples. So if it lasts from N to N, its duration is 1 sample.

File: data2bids.m [hunk 2]
```diff
@@ -1563,7 +1563,7 @@
     end
     % convert to the required fields
     onset     = (begsample-1)/hdr.Fs;
-    duration  = (endsample-begsample+1)/hdr.Fs;
+    duration  = (endsample-begsample)/hdr.Fs;
```
Comment (robertoostenveld) [inline data2bids.m:1566]: Can you please revert this?

**Review by robertoostenveld — CHANGES_REQUESTED** [review]

**Comment by robertoostenveld** [comment]
dank je!

File: data2bids.m (hunk 1 above)
Comment (DidiLamers) [reply data2bids.m]: Okay, done

**Pull request merged**